content/en/sidecar-container.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sidecar Container
+status: Feedback Appreciated
+category: concept
+---
+
+The Sidecar Container is an implementation of the sidecar pattern. 
+An application deployed on a second container will run side by side and share its lifecycle with a primary application running on a main container.
+
+## Problem it addresses
+
+There are scenarios where it is convenient to group [containers](/container/) to deal with problems related to the need to support cross-platform requirements or shared functionality. 
+Problems like logging, monitoring, tracing or [observability](/observability/) in general, and security, traffic management, networking and common configurations applicable to multiple applications. These shared functionalities can be implemented independently as services and run on separate containers.
```

Comment:
```suggestion
Such scenarios include [observability](/observability/) topics like logging, monitoring, tracing, but also security, and networking. 
```

Let's talk more about what the issue is that sidecars are trying to solve. Something like when you have a lot of micorservices it is not so easy and probably also not efficient to handle logging for instance for all of the microservices. You cannot have one microservice collecting the stdout from every other microservices, can you? There are probably more of those examples. 

I deleted the rest because it was not really adding value to the definition IMHO.

Reply:
@iamNoah1 accepted your review and recommendations. Added a couple of examples for the problems. All in commit 5e5ef2e9878569397127c4a065430d562000dce9 .